MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -156,35 +156,31 @@ public void onTracksChanged(EventTime eventTime, TrackGroupArray trackGroups,
   public void onLoadStarted(EventTime eventTime,
       MediaSourceEventListener.LoadEventInfo loadEventInfo,
       MediaSourceEventListener.MediaLoadData mediaLoadData) {
-    bandwidthDispatcher.onLoadStarted(loadEventInfo.dataSpec, mediaLoadData.dataType,
-        mediaLoadData.trackFormat, mediaLoadData.mediaStartTimeMs,
-        mediaLoadData.mediaEndTimeMs, loadEventInfo.elapsedRealtimeMs);
+    bandwidthDispatcher.onLoadStarted(mediaLoadData.mediaStartTimeMs, mediaLoadData.mediaEndTimeMs,
+        loadEventInfo.uri.getPath(), mediaLoadData.dataType, loadEventInfo.uri.getHost());
```

Comment:
Similar to the note below, I imagine `uri` for a `LoadEventInfo` object is always guaranteed to be non-null, right? If not, should we guard this in a check for `null` so we avoid NPEs possibly?

Reply:
added null pointer safeguard.